README.md

```diff
@@ -64,13 +70,7 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
+- Great appreciation to [Microverse](www.microverse.org) staff (Students    Success, Code Reviewer, Coding partners, ..., etc.) 
 ## 📝 License
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
[OPTIONAL] It seems like you forgot to add MIT license to your project. Kindly implement it in your project as well.

Reply:
@Rustamxon7 Thanks for your feedback